python/glow/gwas/lin_reg.py

```diff
@@ -32,30 +32,54 @@ def linear_regression(genotype_df: DataFrame,
     In each Spark task, we project the covariates out of a block of genotypes and then compute the regression statistics for each phenotype,
```

Comment:
I would change "The implementation is based on regenie: https://www.biorxiv.org/content/10.1101/2020.06.19.162354v2" to "the implementation is a distributed version of the method used at regenie..." or something like that.